js/app/readingHistoryModel.js

```diff
@@ -81,8 +81,11 @@ const ReadingHistoryModel= new Knowledge.Class({
     },
 
     mark_article_read: function (article_id) {
+        if (this.is_read_article(article_id))
```

Comment:
Hmm, this looks tidy, but the return value doesn't seem all that useful outside of the context of the property setter in ContentObjectModel. As long as you're calling `is_read_article()` anyway here, I'd say to do it in the property setter instead.